src/System.Text.Json/source_package/README.md

```diff
@@ -1,112 +1,3 @@
-  you might see lots of type conflict warnings (or errors if warnings are treated as errors):
-  - `warning CS0436: The type 'JsonTokenType' in '...' conflicts with the imported type 'JsonTokenType' in 'System.Text.Json, Version=...'. Using the type defined in '...\microsoft.bcl.json.sources\...\JsonTokenType.cs'.`
-
-* You should not reference this package if you are targeting .NET Core 3.0 or higher. If you are multi-targeting, then only conditionally include the package reference.
-  Note that this applies to other package references that you include to fulfill the dependencies of the source package.
-
-### PrivateAssets
-
-* It is recommended that you mark your package reference as private so that this dependency does not accidentally leak outside of your package
-  to applications that might be consuming it (for example, visible within the `deps.json`).
-  ```csproj
-  <PackageReference Include="Microsoft.Bcl.Json.Sources" Version="...">
-     <PrivateAssets>All</PrivateAssets>
-  </PackageReference>
-  ```
-
-## Other Considerations
-
-### InternalsVisibleTo
-
-* Since the source package contains types marked as internal, please be intentional with the use of `InternalsVisibleTo`.
-  It is acceptable to use it where necessary but something we wanted to highlight as an area of consideration.
-
-## Sample Netstandard Library Project File
-
-```csproj
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>latest</LangVersion>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <!-- Suppress warnings for CLSCompliant OR add this attribute-->
-    <!-- <CLSCompliant>true</CLSCompliant> -->
-    <NoWarn>3021</NoWarn>
-    <!-- Suppress warnings for S.T.Json types already defined in .NET Core 3.0, if your TFM is .NET Core 3.0+ (not recommended)
-         OR preferably only conditionally include the package reference -->
-    <!-- <NoWarn>0436</NoWarn> -->
-  </PropertyGroup>
-
-  <!-- If you are multi-targeting and include netcoreapp3.0 in your TFM, make sure this ItemGroup is only condtionally included. -->
-  <!-- Note, the version numbers need to be filled in. -->
-  <ItemGroup>
-    <!-- Do not expose this dependency outside of your package to projects that might be consuming it (i.e. within deps.json). -->
-    <PackageReference Include="Microsoft.Bcl.Json.Sources" Version="...">
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="..." />
-    <PackageReference Include="System.Memory" Version="..." />
-  </ItemGroup>
-
-</Project>
-```
+We are no longer producing the source package, and instead are shipping a System.Text.Json NuGet package which is compatible with netstandard2.0. Please visit [NuGet](https://www.nuget.org/packages/System.Text.Json/) in order to download it.
```

Comment:
@terrajobst, we need to update the metadata regarding this NuGet package since we are going to publish our `System.Text.Json`/replace the previous one:
![image](https://user-images.githubusercontent.com/6527137/56702347-5c4bd500-66b8-11e9-9dbc-accc7f18685f.png)

We should also update the package description here:
https://github.com/dotnet/corefx/blob/51922de496a713568b493e791c60e419042d19b5/pkg/descriptions.json#L1979-L1995

@steveharter, we should add the serializer types there.